Dockerfile

```diff
@@ -1,8 +1,34 @@
 ## Grabbing required binaries for the video processing part
-FROM docker.klink.asia/images/video-processing-cli:0.3.1
+FROM docker.klink.asia/images/video-processing-cli:0.3.1 AS videocli
+# .. we just need this image so we can copy from it
+
+FROM node:8 AS frontend
+WORKDIR /app
+COPY . /app
+RUN \
+    npm install yarn && \
+    yarn config set cache-folder .yarn && \
+    yarn install && \
+    yarn run production
+
+## Installing the dependencies to be used in a later step.
+# Will generate three directories:
+# * /var/www/dms/bin/
+# * /var/www/dms/vendor/
+# * /var/www/dms/public/js/nls/
+FROM php:7.1-fpm AS dependencies
+WORKDIR /var/www/dms
+COPY composer.json composer.lock .
```

Comment:
I believe copying only the composer.json and lock will cause the next step to fail, because you lack the PHP part of the scripts that you want to execute, e.g. install-content-cli is defined in a PHP class that uses some of the composer classes and helpers